docs/api/sandbox-option.md

```diff
@@ -1,9 +1,9 @@
 # `sandbox` Option
 
-> Create a browser window with renderer that can run inside chromium OS sandbox.
+> Create a browser window with a renderer that runs inside the Chromium OS sandbox.
```

Comment:
Thanks for the clarification, that was something that is a little unclear in your documentation. 

I think this is something that needs to be clearly stated up-front. I know people who are using `sandbox: true` under the impression that the Chromium Sandbox is being used.

I'll do another quick edit.